code/datums/ai/generic/find_and_set.dm

```diff
@@ -142,7 +142,11 @@
 /datum/ai_behavior/find_and_set/conscious_person
 
 /datum/ai_behavior/find_and_set/conscious_person/search_tactic(datum/ai_controller/controller, locate_path, search_range)
+	var/list/customers = list()
 	for(var/mob/living/carbon/human/target in oview(search_range, controller.pawn))
 		if(IS_DEAD_OR_INCAP(target) || !target.mind)
 			continue
-		return target
+		customers += target
+
+	if(customers.len)
+		return pick(customers)
```

Comment:
```suggestion
		return pick(customers)
	return null
```

we may as well be explicit about what happens when there are no customers to choose from so consumers of this `find_and_set` know for a fact that they can be getting nulls.